scenarios/monitoring/cdp_monitoring/scripts/ingest_ps_configuration.py

```diff
@@ -0,0 +1,37 @@
+import requests
+import pandas as pd
+import pytd
+import os
+import json
+
+def convert_to_json(s):
+  return json.dumps(s)
+
+def get_all_parent_segment_configuration(url, headers):
+    print(url)
+    res = requests.get(url=url, headers=headers)
+    if res.status_code != requests.codes.ok:
+        res.raise_for_status()
+    
+    data = res.json()
+    for d in data:
+        for k in d.keys():
+            if type(d[k]) is dict:
+                d[k] = json.dumps(d[k])
+    
+    return data
+
+def run(session_unixtime, dest_db, dest_table, api_endpoint='api.treasuredata.com', cdp_api_endpoint='api-cdp.treasuredata.com'):
+    url = 'https://%s/audiences' % cdp_api_endpoint
+    headers = {'Authorization': 'TD1 %s' % os.environ['TD_API_KEY']}
+    l = get_all_parent_segment_configuration(url, headers)
+    if len(l) == 0:
+      print('no import record')
+      return
+    df = pd.DataFrame(l)
+    df['time'] = int(session_unixtime)
+    df['attributes'] = df['attributes'].apply(convert_to_json)
+    df['behaviors'] = df['behaviors'].apply(convert_to_json)
```

Comment:
Does the API return schedule and query engine (Presto and Hive, and Hive only for example)?

Reply:
Unfortunately, to obtain detailed settings, the ParentSegment ID must be specified.
So, we will need to issue as many REST APIs as there are ParentSegments.
https://api-docs.treasuredata.com/pages/audience_api_v1/operation/audiences_show/